docs/interpreter/spark.md

```diff
@@ -145,6 +145,11 @@ You can also set other Spark properties which are not listed in the table. For a
     <td>true</td>
     <td>Do not change - developer only setting, not for production use</td>
   </tr>
+  <tr>
+  <td>zeppelin.spark.uiWebUrl</td>
+    <td></td>
+    <td>Override Spark UI default URL</td>
```

Comment:
could you add a bit more on what are the condition to set this, example value etc..?